### DISCUSSION TITLE
Supernote - todo sync with Obsidian

### DISCUSSION THREAD

**Discussion by PapaHawk14** [philips/supernote-obsidian-plugin#30] [Ideas]
Hi There
Great plugin, thank you - it's been the realisation of what I wanted in a note-taking tool integrated into my Obsidian Vault.

What are your thoughts on extending the plugin to bring the newly released ToDo data to Obsidian?
It would be very helpful, and I expect I wouldn't be the only one asking :-)

Cheers Grant

**Comment by philips** [comment]
Is there a spec for the todo system?

**Comment by PapaHawk14** [comment]
No Not that I have seen. But I'm not a developer engaged with them so
likely if its available I wouldnt see it sorry.

Thanks for your response.

On Fri, Jun 7, 2024 at 9:08 AM Brandon Philips ***@***.***>
wrote:

> Is there a spec for the todo system?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/philips/supernote-obsidian-plugin/discussions/30#discussioncomment-9694916>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVHPCVYSCCJE3UDTNNXIVELZGDXIDAVCNFSM6AAAAABI5UMIT6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TMOJUHEYTM>
> .
> You are receiving this because you authored the thread.Message ID:
> <philips/supernote-obsidian-plugin/repo-discussions/30/comments/9694916@
> github.com>
>


-- 
==========================
Grant Hawksworth
Adelaide Sth Australia
email: ***@***.***
